recipes/libffi.recipe

```diff
@@ -20,17 +20,18 @@ FFI_H_UNVERSAL='''\
 
 class Recipe(recipe.Recipe):
     name = 'libffi'
-    version = '3.2.1'
+    version = '3.1.1'
     autoreconf = True
-    commit = '20562ac0427c3578250d04c6e34fb0127d4551cf'
+    commit = 'frida/master'
```

Comment:
Let's use the a fixed commit here as well: 4e2166049f897a904e4c3ce367e4e8ddb54e0ad0